.github/workflows/docker-rspec.yml

```diff
@@ -0,0 +1,45 @@
+---
+name: Rspec within Docker
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    name: Run rspec test suite inside Docker
+    runs-on: ubuntu-latest
+    env:
+      GOVUK_NOTIFY_API_KEY: ${{ secrets.GOVUK_NOTIFY_API_KEY }}
```

Comment:
The intent was for this to sit in the credentials file (along with any other API keys / settings).  Then only the RAILS_MASTER_KEY is needed to unlock that file.

Reply:
That's what I thought. At the time I couldn't get into the creds so I was passing in my own Notify test key. I am now passing in the master key.